step-capstone/src/scripts/DragTravelObject.js

```diff
@@ -0,0 +1,56 @@
+import _, { clone } from "lodash";
+export const ItemTypes = {
+  TRAVELOBJECT: "travelobject",
+};
+
+const minPerDiv = 30.0;
+const pixelPerDiv = 47.5;
+export const moveTravelObject = (item, monitor) => {
+  var diffPos = window.event.clientY - item.y;
+  // Handle Drag and drop
+  var cloneData = _.cloneDeep(item.data);
+  const startOfDisplayDate = new Date(
+    item.displayDate.getFullYear(),
+    item.displayDate.getMonth(),
+    item.displayDate.getDate(),
+    0,
+    0,
+    0
+  );
+
+  const endOfDisplayDate = new Date(
+    item.displayDate.getFullYear(),
+    item.displayDate.getMonth(),
+    item.displayDate.getDate(),
+    23,
+    59,
+    59
+  );
+  var newStartDate = new Date(
+    item.data.startDate.getTime() +
+      (((diffPos) * minPerDiv) / pixelPerDiv) *
+        60000
```

Comment:
Nit: This calculation is a little confusing, what's the 60,000? Maybe add an explanatory comment or make it clearer by using additional named variables.

Reply:
Thank you, I have addressed this confusion in the latest commit.